content/messaging/agent-chat-api/rtm-reference/index.mdx

```diff
@@ -425,39 +425,64 @@ The **Message** event contains text message to other chat users.
-| `elements.title`                 | optional  |                                                                     |
-| `properties`                     | optional  | The [properties](#properties) object                                |
+Rich message (RM) event contains a rich message data structure. [Read more](/extending-ui/extending-chat-widget/rich-messages/) about rich messages.
+
+### Request
+
+| Field                            | Required | Data type | Notes                                                                                             |
+| -------------------------------- | -------- | --------- | ------------------------------------------------------------------------------------------------- |
+| `custom_id`                      | no       | `string`  | You can give your RM a custom ID.                                                                 |
+| `type`                           | yes      | `string`  | Event type: `rich_message`                                                          |
+| `recipients`                     | yes      | `string`  | Those who can display the rich message: `all` (default) or `agents`                               |
+| `properties`                     | no       | `object`  | The [properties](#properties) data structure                                                              |
+| `template_id`                    | yes      | `string`  | Defines how every Rich Message will be presented. Values: `card` or `quick_replies`.              |
+| `elements`                       | no       | `array`   | Can contain up to 11 `element` objects.                                                           |
+| `elements.title`                 | no       | `string`  | Displays formatted text on RMs.                                                                   |
+| `elements.subtitle`              | no       | `string`  | Displays formatted text on RMs.                                                                   |
+| `elements.image`                 | no       | `image`   | Displays images on RMs. Optional params: `name`, `url`, `content_type`, `size`, `width`, `height`.|
+| `elements.buttons`               | no       | `array`   | Displays buttons. Can contain up to 11 `button` objects.                                          |
+| `elements.buttons.type`          | no       | `array`   | Defines the behavior after a user clicks the button. Required if both `elements.buttons` and `elements.buttons.value` are present. Possible values: `webview`, `message`, `url`, `phone`. [More...](/extending-ui/extending-chat-widget/rich-messages/#getting-started)  |
+| `elements.buttons.text`          | no       | `string`  | Required if `elements.buttons` is present.                                                        |
+| `elements.buttons.value`         | no       | `string`  | Required if both `elements.buttons` and `elements.buttons.type` are present.                      |
```

Comment:
we can remove this fragment: `Required if both elements.buttons...` just leave "required if `elements.buttons.type` is present"

Reply:
We use form `Should be used together with` in message event description, maybe we should use it here as well?